.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+packages
```

Comment:
Ok, I see where you're coming from. I see a member of a monorepo semantically related to the other packages in the monorepo, and thus should remain as DRY as possible. As long as asini continues support both strategies, I'm fine with these changes for asini itself.